my_cinema_project/cinema/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from cinema.views import movie_view
+
+urlpatterns = [
+    path("movies/", movie_view, name="movie-list"),
+    path("movies/<int:pk>/", movie_view, name="movie-update"),
+    path("movies/<int:pk>/", movie_view, name="movie-detail"),
+    path("movies/", movie_view, name="movie-create"),
+    path("movies/<int:pk>/", movie_view, name="movie-delete")
```

Comment:
There is no need to register the same endpoint three times with different names, leave only `movie-detail`

Reply:
Should I remove the url with "movie-create"